---
abstract: Researchers use ChIP binding data to identify potential transcription factor binding sites. They use gene expression data from sequencing or microarrays to quantify the effect of the factor over-expression or knockdown on its targets. The integration of the binding and expression data therefore can be used to improve the understanding of a transcription factor function. In this workshop, I present a complete workflow for integrating the gene expression (microarray/RNA-seq) and DNA-binding data (ChIP-seq) to predict the combined function of two transcription factors using R/Bioconductor. The example we will be using in the workshop is from real datasets of two functionally and evolutionary-related transcription factors YY1 and YY2 in HeLa cells. We will try to identify the factor-specific and the shared targets of the factors in this particular cell line. Then we will use a technique to find out the aggregate functions of the factors on their individual (inducer or repressor) and common targets (cooperative or competitive).
address:
  city: Virtual
  country: ""
  postcode: ""
  region: ""
  street: ""
all_day: false
authors: []
date: "2021-02-27T10:30:00Z"
date_end: ""
event: R4Bioinfo
event_url: https://www.meetup.com/rladies-tunis/events/276421511/
featured: true
image:
  caption: ""
  focal_point: Right
links:
- icon: meetup
  icon_pack: fab
  name: meetup
  url: "https://www.meetup.com/rladies-tunis/events/276421511/"
location: Virtual
projects:
- transcription-regulation
publishDate: "2021-02-27T10:30:00Z"
slides: ""
summary: ""
tags: []
title: Integrating gene expression and DNA-binding data using R/Bioconductor

url_code: https://github.com/MahShaaban/IntegratingDataBioc
url_pdf: ""
url_slides: ""
url_video: "https://www.youtube.com/watch?v=9cYdBwLsr9A"
---

- This workshop is scheduled for February, 27 and is hosted by
[R-Ladies Tunisia](https://rladies.org/tunisia-rladies/) 
as part of the
[R4Bioinfo](R4Bioinfo) series. [(Event Page).](https://www.meetup.com/rladies-tunis/events/276421511/)
- Part of the materials was presented at 
[EuropBioc 2021](https://github.com/MahShaaban/targetShop). 
- The code is organized in a 
[BiocWorkshop](https://github.com/seandavi/BuildABiocWorkshop) format.

## Overview

Researchers use ChIP binding data to identify potential transcription factor binding sites. They use gene expression data from sequencing or microarrays to quantify the effect of the factor over-expression or knockdown on its targets. The integration of the binding and expression data therefore can be used to improve the understanding of a transcription factor function. In this workshop, I present a complete workflow for integrating the gene expression (microarray/RNA-seq) and DNA-binding data (ChIP-seq) to predict the combined function of two transcription factors using R/Bioconductor. The example we will be using in the workshop is from real datasets of two functionally and evolutionary-related transcription factors YY1 and YY2 in HeLa cells. We will try to identify the factor-specific and the shared targets of the factors in this particular cell line. Then we will use a technique to find out the aggregate functions of the factors on their individual (inducer or repressor) and common targets (cooperative or competitive).
